### ISSUE TITLE
Unintuitive Mindbreaker +45 Magic Damage to buildings

### ISSUE THREAD

**Issue by dmkdmn** [ValveSoftware/Dota2-Gameplay#20884]
### Item name

Mindbreaker

### Description

As of patch 7.37, Mindbreaker provides a flat damage buff to buildings.  

Situation: 

* Omniknight with aghanim's scepter casts guardian angel on a building.  
* Hero equipped with Mindbreaker no longer affects building with base damage, but magic damage from Mindbreaker goes through.  It's full damage because all buildings have 0% magic resistance. 
* See attached match id.

### Example Match ID and Timestamp

7891260274 1:03:18

**Comment by zxcshadowfriend** [comment]
yeah thats how its supposed to work... guardian angel only protects from physical damage

**Comment by dmkdmn** [comment]
It works that way for heroes or creeps, sure, but I'm talking specifically buildings.

Dragon Scale, meteor hammer, and nether blast are a handful of the items that provide direct magic damage to buildings and they're all called out specifically. No other +magic attack damage items do this - maelstrom doesn't, Muerta's Ult doesn't, etc. 

It seems unintended especially in the context of other neutral items/skills that specifically call out that the magic damage works on buildings.
